Examples/SwiftSyntax.md

```diff
@@ -0,0 +1,52 @@
+# Swift Syntax Examples
```

Comment:
This just duplicates the code that’s in `AddOneToIntegerLiteral.swift`.

I think we can just include the non-code parts as comments in `AddOneToIntegerLiteral.swift` and delete this file.